adventure/api.py

```diff
@@ -59,9 +60,13 @@ def move(request):
         players = room.playerNames(player_uuid)
         return JsonResponse({'name':player.user.username, 'title':room.title, 'description':room.description, 'players':players, 'error_msg':"You cannot move that way."}, safe=True)
 
-
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
     # IMPLEMENT
+    player = request.user.player
+    player_id = player.id
+    player_uuid = player.uuid
+    data = json.loads(request.body)
+    pusher.trigger(f'p-channel-{player_uuid}', u'broadcast', f'{data}')
```

Comment:
You actually want to push to every `player_uuid` that is in the current room.
```
def say(request):
    player = request.user.player
    player_id = player.id
    data = json.loads(request.body)
    message = data['message']
    room = player.room()
    currentPlayerUUIDs = room.playerUUIDs(player_id)
    for p_uuid in currentPlayerUUIDs:
            pusher.trigger(f'p-channel-{p_uuid}', u'broadcast', {'message': f'{player.user.username} says {message}.'})
    return JsonResponse({'response':f"Success. You said {message}"}, safe=True, status=200)
```

Reply:
Thank you.